go.mod

```diff
@@ -1,18 +1,36 @@
 module github.com/bitrise-steplib/steps-google-play-deploy
 
-go 1.16
+go 1.20
```

Comment:
I think I have asked this question in someone else's PR already but I do not remember the answer and could not find which PR it was. 

The default go on our stacks is 1.18 and here we are defining a more recent version. The tests pass and everything looks fine so it does not cause a problem right but technically this value should point the minimum version the app is compatible I think. So just want to double check that this is fine here.

Reply:
It does not cause issues in this case. However in and after go 1.21, go runtime takes extra steps to use the defined go version, so will have to be more careful.